framework/src/modules/network/components/storage/entities/index.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+module.exports = {
+	Peer: require('./peer'),
+	Migration: require('./migration'),
```

Comment:
can we actually register 2 different migration?

Reply:
now it's only registered once in chain module